sktime/classification/kernel_based/_rocket_classifier.py

```diff
@@ -30,15 +30,17 @@ class RocketClassifier(_DelegatedClassifier):
     Shorthand for the pipeline
     `rocket * StandardScaler(with_mean=False) * RidgeClassifierCV(alphas)`
     where `alphas = np.logspace(-3, 3, 10)`, and
-    where `rocket` depends on params `rocket_transform`, `use_multivariate` as follows:
-
-        | rocket_transform | `use_multivariate` | rocket (class)          |
-        |------------------|--------------------|-------------------------|
-        | "rocket"         | any                | Rocket                  |
-        | "minirocket"     | "yes               | MiniRocketMultivariate  |
-        | "minirocket"     | "no"               | MiniRocket              |
-        | "multirocket"    | "yes"              | MultiRocketMultivariate |
-        | "multirocket"    | "no"               | MultiRocket             |
+    where `rocket` depends on params `rocket_transform`, `use_multivariate` as follows
+
+    ================ ================== =======================
+    rocket_transform `use_multivariate` rocket (class)
+    ================ ================== =======================
+    "rocket"         any                Rocket
+    "minirocket"     "yes               MiniRocketMultivariate
```

Comment:
For consistency, I suppose it should have a closing double quote.
```suggestion
    "minirocket"     "yes"               MiniRocketMultivariate
```